libraries/botbuilder-dialogs/src/converter.ts

```diff
@@ -9,12 +9,12 @@
 /**
  * The converter converts object from one type to another.
  */
-export interface Converter<From = any, To = any> {
+export interface Converter<From = unknown, To = unknown> {
     convert(value: From): To;
 }
 
 export type Converters<T> = {
-    [K in keyof Partial<Omit<T, 'converters' | 'configure'>>]:
-        | Converter<any, T[K]>
-        | ((...args: any[]) => Converter<any, T[K]>);
-}
+    [K in keyof Partial<T>]: T[K] extends Function
```

Comment:
This is a step in the right direction, I think there are two types coupled here though. In the interested of separation of concerns they should be distinct.

The first type is `Properties<T>` that omits all functions from a type.

The second is the `Converters<T>` that expects the type parameter to contain the proper types. Users would then write `Converter<Properties<T>>`.